### ISSUE TITLE
investigate a11y guidelines regarding color contrast non default states

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#1009]
```[tasklist]
### Tasks
- [x] what should be the color contrast between the body background and for example a dropdown panel, how do we handle this?
- [x] gather resources
- [x] define our solution
- [x] important visual cues should have a color contrast of 1:3
- [x] update popover/select borders + box shadow
```
